README.md

```diff
@@ -1,6 +1,6 @@
 # baad.data
 
-[![Build Status](https://travis-ci.org/traitecoevo/baad.data.png?branch=master)](https://travis-ci.org/traitecoevo/baad.data)
+[![Build Status](https://travis-ci.org/the-Hull/baad.data.svg?branch=master)](https://travis-ci.org/the-Hull/baad.data)
```

Comment:
Can we change this back to traitecoevo?

Reply:
Sorry for the late follow up @dfalster 
I'll get to this this week, most likely! 
Cheers!

Reply:
Sorry for the delay - I will try to squash the commits!